.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -4,41 +4,70 @@ about: Create a report to help us improve TypeScript
+### 🔎 Search Terms
 
-Please fill in the *entire* template below.
+<!--
+  What search terms did you use when trying to find an existing bug report?
+  List them here so people in the future can find this one more easily.
 -->
 
-<!--
-Please try to reproduce the issue with the latest published version. It may have already been fixed.
+### 🕗 Version & Regression Information
+
+<!-- When did you start seeing this bug occur?
+
+"Bugs" that have existed in TS for a long time are very likely to be FAQs; refer to
+  https://github.com/Microsoft/TypeScript/wiki/FAQ#common-bugs-that-arent-bugs
+
+If possible, please try testing the nightly version of TS to see if it's already been fixed.
 For npm: `typescript@next`
 This is also the 'Nightly' version in the playground: http://www.typescriptlang.org/play/?ts=Nightly
+
+Note: The TypeScript Playground can be used to try older verions of TypeScript.
+
+Please keep and fill in the line that best applies:
 -->
-**TypeScript Version:**  3.7.x-dev.201xxxxx
+ - This is a crash
+ - This changed between versions ______ and _______
+ - This is the behavior in every version I tried, and I reviewed the FAQ for entries about _________
+ - I was unable to test this on prior versions because _______
+
+### ⏯ Playground Link
 
-<!-- Search terms you tried before logging this (so others can find this issue more easily) -->
-**Search Terms:**
+<!--
+  A link to a TypeScript Playground "Share" link which shows this behavior
+
+  The TypeScript Workbench can be used for more complex setups, try
+  https://www.typescriptlang.org/dev/bug-workbench/
+
+  As a last resort, you can link to a repo, but these will be slower for us to investigate.
+-->
+[Playground link with relevant code](https://www.typescriptlang.org/play?#code/PTAEFkE9QYwewCYFNQHM5IM6gBZIE5JA)
 
-**Code**
+### 💻 Code
 
+<!-- Please post the relevant code sample here as well-->
 ```ts
-// A *self-contained* demonstration of the problem follows...
-// Test this by running `tsc` on the command-line, rather than through another build tool such as Gulp, Webpack, etc.
+// We can quickly address your report if:
+//  - The code sample is short. Nearly all TypeScript bugs can be demonstrated in 20-30 lines of code!
+//  - It doesn't use external libraries. These are often issues with the type definitions rather than TypeScript bugs.
+//  - The incorrectness of the behavior is readily apparent from reading the sample.
+// Reports are slower to investigate if:
+//  - We have to pare too much extraneous code.
+//  - We have to clone a large repo and validate that the problem isn't elsewhere.
+//  - The sample is confusing or doesn't clearly demonstrate what's wrong.
 ```
 
-**Expected behavior:**
+### 🙂 Expected behavior
 
-**Actual behavior:**
+<!-- What you expected to happen, and why -->
 
-**Playground Link:** <!-- A link to a TypeScript Playground "Share" link which demonstrates this behavior -->
+### 🙁 Actual behavior
```

Comment:
I actually kind of always disliked the ordering because what I tend to do is follow the template, and I want to just paste in the existing behavior. That means I have to jump down to the "actual behavior" section first, massage that, and then copy/paste that into the "expected behavior" section. I'd love if we were open to switching to "Current Behavior" followed by "Expected Behavior" or "Proposed Behavior"

Reply:
Agree; swapped